src/com/sun/ts/lib/harness/keyword.properties

```diff
@@ -57,21 +57,39 @@ com/sun/ts/tests/appclient/deploy/ejblink/single = javaee_optional
 com/sun/ts/tests/appclient/deploy/ejbref/single = javaee_optional
 
 com/sun/ts/tests/assembly = assembly javaee
+
 com/sun/ts/tests/connector = connector javaee connector_standalone connector_web_profile javaee_web_profile_optional
 com/sun/ts/tests/connector/deployment = javaee
 com/sun/ts/tests/connector/localTx/msginflow = connector_mdb_optional javaee
-com/sun/ts/tests/connector/localTx/transaction = connector connector_jta_optional javaee
```

Comment:
Should RMI_IIOP is not really optional for Full Platform, should we remove `javaee javaee_optional`?

```
com/sun/ts/tests/connector/localTx/transaction = connector connector_jta_optional rmi_iiop_optional
```
I'm not even sure ^ is right as previously we ran this test if `connector` was specified and still do, is that right?  Would the following be better?

```
com/sun/ts/tests/connector/localTx/transaction = connector_jta_optional rmi_iiop_optional
```
So that this test is only run if either of `connector_jta_optional rmi_iiop_optional` were specified?